### DISCUSSION TITLE
Can you upload documents in a folder???

### DISCUSSION THREAD

**Discussion by akim67212** [community/community#138311] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hey, I was just wondering, once you create a folder in your repository, are you able to upload documents into that folder? If so, how??

**Comment by queenofcorgis** [comment, marked as answer]
@akim67212 yes - once you create a folder you can upload files directly into the folder.  Click into the folder you'd like to upload files into and on the righthand side use the Add File button to select `Upload Files`

![Screenshot 2024-09-12 at 10 09 45 AM](https://github.com/user-attachments/assets/4f86f71e-495d-4852-94b3-78bc56635c04)

This document walks you through the steps of uploading [documents](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository), this can be applied at the folder level as well 😄

**Reply by akim67212** [reply]
@queenofcorgis Thank you so much!

**Reply by queenofcorgis** [reply]
You're welcome, thanks for marking the answer so this knowledge can help others 🌟
